spec/models/user_spec.rb

```diff
@@ -11,10 +11,10 @@
     it { is_expected.to have_many(:events) }
   end
 
-  it 'should only send one email during creation' do
+  it 'does not send an email when #create is called' do
```

Comment:
It is very related, and it was intentional. As mentioned in my commit message, Devise automatically calls `send_confirmation_instructions` when the user is created (via a model callback). Unfortunately, Devise does not make it possible to send it additional parameters that can be included in the email. Therefore, I had to create a new `send_custom_confirmation_instructions` method that accepts an argument, and at the same time override Devise's `send_confirmation_instructions` to do nothing. Otherwise, the user would receive 2 emails.